JPS_Ontology/ontology/ontoubemmp/ontoubemmp.csv

```diff
@@ -3,45 +3,37 @@ OntoUBEMMP,TBox,https://www.theworldavatar.com/kg/ontoubemmp/,https://www.w3.org
+ElectricitySupply,Class,EnergySupply,IS-A,,,,The amount of electricity energy produced,https://www.theworldavatar.com/kg/ontoubemmp/,
 AnnualElectricitySupply,Class,ElectricitySupply,IS-A,,,,,https://www.theworldavatar.com/kg/ontoubemmp/,
 ElectricityConsumption,Class,EnergyConsumption,IS-A,,,,The amount of electricity consumed by appliances and lighting,https://www.theworldavatar.com/kg/ontoubemmp/,
 AnnualElectricityConsumption,Class,ElectricityConsumption,IS-A,,,,,https://www.theworldavatar.com/kg/ontoubemmp/,
 ThermalConsumption,Class,EnergyConsumption,IS-A,,,,The amount of thermal energy consumed by a heating/cooling system,https://www.theworldavatar.com/kg/ontoubemmp/,
 AnnualThermalConsumption,Class,ThermalConsumption,IS-A,,,,,https://www.theworldavatar.com/kg/ontoubemmp/,
 GridConsumption,Class,EnergyConsumption,IS-A,,,,The total grid electricity energy consumption,https://www.theworldavatar.com/kg/ontoubemmp/,
 AnnualGridConsumption,Class,GridConsumption,IS-A,,,,,https://www.theworldavatar.com/kg/ontoubemmp/,
-hasDevice,Object Property,,,http://www.purl.org/oema/infrastructure/Building ,http://www.purl.org/oema/enaeq/Device,,,https://www.theworldavatar.com/kg/ontoubemmp/,
-http://www.purl.org/oema/enaeq/consumesEnergy,Object Property,,,http://www.purl.org/oema/enaeq/EnergyConsumptionDevice UNION http://www.purl.org/oema/infrastructure/Building ,EnergyConsumption,,,https://www.theworldavatar.com/kg/ontoubemmp/,
-https://www.auto.tuwien.ac.at/downloads/thinkhome/ontology/EnergyResourceOntology.owl#producesEnergy,Object Property,,,http://www.purl.org/oema/enaeq/EnergyGenerator,https://www.auto.tuwien.ac.at/downloads/thinkhome/ontology/EnergyResourceOntology.owl#EnergySupply,,,https://www.theworldavatar.com/kg/ontoubemmp/,
-hasArea,Object Property,,,http://www.purl.org/oema/enaeq/Device,http://www.ontology-of-units-of-measure.org/resource/om-2/Area,,,https://www.theworldavatar.com/kg/ontoubemmp/,
-SolarCollectors,Class,SolarEnergyGenerator,IS-A,,,,Solar collector: converts solar radiation into usable thermal energy,https://www.theworldavatar.com/kg/ontoubemmp/,
+consumesEnergy,Object Property,,,https://www.theworldavatar.com/kg/ontodevice/Device UNION https://w3id.org/bot#Building ,EnergyConsumption,,A relationship between a facility and its consumed energy,https://www.theworldavatar.com/kg/ontoubemmp/,
+producesEnergy,Object Property,,,SolarDevice,EnergySupply,,A relationship that links the solar device to the energy it produces,https://www.theworldavatar.com/kg/ontoubemmp/,
+hasArea,Object Property,,,https://www.theworldavatar.com/kg/ontodevice/Device,http://www.ontology-of-units-of-measure.org/resource/om-2/Area,,,https://www.theworldavatar.com/kg/ontoubemmp/,
+SolarCollectors,Class,SolarDevice,IS-A,,,,Solar collector: converts solar radiation into usable thermal energy,https://www.theworldavatar.com/kg/ontoubemmp/,
```

Comment:
Wouldn't it be less confusing to use singular `SolarCollector`, i.e. removing the 's' from this and related classes? The comment uses singular...

Reply:
Are you sure you don't want to change this?

Reply:
Sorry, my mistake. I missed this out previously. Thank you for catching it :)